Sources/BraintreeCore/BTShopperInsightsClient.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+///  Use `BTShopperInsightsClient` to optimize your checkout experience by prioritizing the customer’s preferred payment methods in your UI.
+///  By customizing each customer’s checkout experience, you can improve conversion, increase sales/repeat buys and boost user retention/loyalty.
+///  - Note: This feature is in beta. It's public API may change in future releases.
+@objcMembers class BTShopperInsightsClient: NSObject {
+    
+    private let apiClient: BTAPIClient
+    
+    /// Creates a `BTShopperInsightsClient`
+    /// - Parameter apiClient: A `BTAPIClient` instance.
+    /// - Note: This features only works with a client token.
+    public init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+    }
+    
+    /// This method confirms if the customer is a user of PayPal services using their email and phone number.
+    /// - Parameters:
+    ///   - email: The buyer's email address
+    ///   - phone: The buyer's phone number
+    /// - Returns: A `BTShopperInsightsResult` instance
+    /// - Note: This feature is in beta. It's public API may change in future releases.
+    public func getRecommendedPaymentMethods(request: BTShopperInsightsRequest) async throws -> BTShopperInsightsResult {
```

Comment:
Do we want a completion method for this as well similar to the other clients or just this one?

Reply:
I intentionally left it out to use this as a case for gauging merchant feedback on if we need completion handler syntax. Figured let's start with less & add based on need?

Also this feature is going out as a beta first so we can (and will) iterate. What do you think?

Reply:
I think that makes sense - will be a good way to gauge things for potentially removing completion syntax in v7. Let's do it!